nitric/apis/options.go

```diff
@@ -12,14 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package nitric
-
-import (
-	"github.com/nitrictech/go-sdk/handler"
-)
+package apis
 
 type (
 	ApiOption    = func(api *api)
+	RouteOption  = func(route Route)
 	MethodOption = func(mo *methodOptions)
```

Comment:
We should probably use types here instead of Aliases to make the required type clearer in intellisense:

e.g.

![image](https://github.com/user-attachments/assets/90aa8331-ac23-4064-811d-0dd61039733d)

when compared to:

![image](https://github.com/user-attachments/assets/00b8b15f-0bac-44b8-a136-4e2448d09486)